src/chocolatey/infrastructure.app/services/INugetService.cs

```diff
@@ -49,5 +51,11 @@ public interface INugetService : ISourceRunner
         /// </summary>
         /// <param name="packageName">Name of the package.</param>
         void remove_rollback_directory_if_exists(string packageName);
+
+		/// <summary>

```

Comment:
Looks like this method is bumped over an extra tab as well, i.e. it isn't inline with the other methods.

Reply:
GitHub by default shows tabs as 8 spaces instead of 4 add ?ts=4 to file view url, it doesn't work with diffs.
Anyway I will convert tabs to spaces.

Reply:
Sounds good, thanks!